tensorflow/core/util/gpu_solvers.h

```diff
@@ -31,13 +31,10 @@ limitations under the License.
 #include "third_party/gpus/cuda/include/cuda.h"
 #include "third_party/gpus/cuda/include/cusolverDn.h"
 #else
-#include "rocm/rocm_config.h"
```

Comment:
This line should be reinstated because it is needed for the TF_ROCM_VERSION macro.